app/views/home/temp_dashboard.html.erb

```diff
@@ -0,0 +1,22 @@
+<div id="dashboard">
+  <div id="page">
+    <div class="banner">
+      <iron-icon icon="icons:error" class="alert"></iron-icon>
+ You've been assigned as an author on a new paper. <a href="#">Begin review.</a>
```

Comment:
Good catch. As for whether to show this - perhaps we should just show this if they have an assigned paper that they haven't commented on?